### PR TITLE
frontend get request implementation, App restructuring and styling

### LINKED ISSUE #14: [Priority] Working API call to Surgeo/Flask backend to fetch BisG predictions
after a working API route for BisG/Surgeo is finished in the flask backend, work on setting up the react frontend to get the webapp to properly call and access the bisg predictions.

### PR DESCRIPTION
**Pull request by austinwli**
frontend design/surgeo api-call. form only takes surname

### CODE REVIEW

File: react-frontend-app/src/App.js [hunk 1]
```diff
@@ -13,61 +16,60 @@ import Display from './components/Display'
...
-        <form onSubmit={this.handleSubmit}>
-            <label>
-              Insert Form Information:
-            <input type="text" value={this.state.input_val} onChange={this.handleChange}/>
-            </label>
-            <input type="submit" value="Submit" />
-        </form>
-
-        <Display />
+    // axios.get(`/surgeo?surname=${surname}&zipcode=${zipcode}`)
```
Comment (dzheng21) [inline react-frontend-app/src/App.js]: remove comments when finished with everything

Comment (dzheng21) [reply react-frontend-app/src/App.js]: unless they're descriptive/useful, but in this case seems like it's like old code

File: react-frontend-app/src/App.js [hunk 2]
```diff
@@ -13,61 +16,60 @@ import Display from './components/Display'
...
+    //   .then(res => console.log(res));
+  }
 
-        <Footer />
+  return (
+    <div className="App">
+      {/* Still in progress */}
+      {/* <MainNavBar /> */}
+      {/* <Description /> */}
+      <br/><br/><br/><br/><br/>
```
Comment (dzheng21) [inline react-frontend-app/src/App.js:48]: There may be better ways to position things than breaks, but we can play w/ structure and formatting later on

File: react-frontend-app/src/App.js [hunk 3]
```diff
@@ -13,61 +16,60 @@ import Display from './components/Display'
...
+      <h1>Zest AI Race Predictor Prototype</h1>
+      <br/>
+      <form action="" onSubmit={handleSubmit}>
```
Comment (dzheng21) [inline react-frontend-app/src/App.js]: i think i'd remove the action attribute if it does nothing here, just for consistency and to reduce confusion

File: src/app.py [hunk 4]
```diff
@@ -12,13 +13,12 @@
 def say_hello_world():
     return {'result': "Flask says Hello World"}
 
-@app.route('/surgeo', methods=["GET"])
+@app.route('/surgeo', methods=["GET", "POST"])
```
Comment (dzheng21) [inline src/app.py:16]: Just curious, why did you add Post here? I actually think Post is not a bad idea for this endpoint (there are a few differences btwn get and post). We can table this discussion to another time, good to note.

**Review by dzheng21 — COMMENTED** [review]
good overall, minor fixes with cleaning up code/making future mods and we can merge!

**Review by dzheng21 — APPROVED** [review]

**Review by kaylanhua — APPROVED** [review]
Looks good to me!
